coordinator/src/block_streams_handler.rs

```
@@ -62,9 +62,7 @@ impl BlockStreamsHandlerImpl {
             .clone()
             .stop_stream(Request::new(request.clone()))
             .await
-            .map_err(|e| {
-                tracing::error!(stream_id, "Failed to stop stream\n{e:?}");
-            });
+            .context(format!("Failed to stop stream: {stream_id}"))?;
```

Propagate the error up, don't swallow, so we can handle it in `Synchroniser`.